src/test/cli/rbd/help.t

```diff
@@ -78,16 +73,13 @@ Skip test on FreeBSD as it generates different output there.
                                         pool.
       mirror pool status                Show status for all mirrored images in
                                         the pool.
-      nbd list (nbd ls)                 List the nbd devices already used.
-      nbd map                           Map image to a nbd device.
-      nbd unmap                         Unmap a nbd device.
       object-map check                  Verify the object map is correct.
       object-map rebuild                Rebuild an invalid object map.
       pool init                         Initialize pool for use by RBD.
       remove (rm)                       Delete an image.
       rename (mv)                       Rename image within pool.
       resize                            Resize (expand or shrink) image.
-      showmapped                        Show the rbd images mapped by the kernel.
+      showmapped                        Show mapped rbd images.
```

Comment:
The ```showmapped``` command has always seemed oddly named to me. Perhaps since we are changing/merging, we can just create new "device map", "device list", and "device unmap" commands and deprecate all the older CLI actions(?)

Reply:
@dillaman I like  "device map|list|unmap"  too. I was reluctant to change (depricate) krbd commands though as in contrast to nbd/ggate they have long history, mentioned in many docs, and are very likely used in many scripts.

But if you and other think it is ok I will change this.

Reply:
... I think if we keep it deprecated for two releases, that will leave plenty of time to adjust.